schemas/iso19139/src/main/plugin/iso19139/loc/eng/labels.xml

```diff
@@ -1941,8 +1941,8 @@
 			<option value="OGC:WFS-1.0.0-http-get-capabilities">OGC-WFS Web Feature Service (ver 1.0.0)
 			</option>
 			<option value="OGC:WFS-G">OGC-WFS-G Gazzetteer Service</option>
-			<option value="OGC:WMC-1.1.0-http-get-capabilities">OGC-WMC Web Map Context (ver 1.1)
-			</option>
+			<option value="OGC:WMC-1.1.0-http-get-capabilities">OGC-WMC Web Map Context (ver 1.1)</option>
```

Comment:
Drop that. We should keep simple protocols only without version and ops details.